common/djangoapps/entitlements/tests/factories.py

```diff
@@ -1,18 +1,20 @@
 import string
-import uuid
+from uuid import uuid4
 
 import factory
 from factory.fuzzy import FuzzyChoice, FuzzyText
 
 from entitlements.models import CourseEntitlement
 from student.tests.factories import UserFactory
+from course_modes.helpers import CourseMode
 
 
 class CourseEntitlementFactory(factory.django.DjangoModelFactory):
     class Meta(object):
         model = CourseEntitlement
 
-    course_uuid = uuid.uuid4()
-    mode = FuzzyChoice(['verified', 'profesional'])
+    uuid = uuid4()
```

Comment:
This will result in every `CourseEntitlement` having the same value for this field. Use `uuid = factory.LazyFunction(uuid4)`. Do the same for the `course_uuid` field.